catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -122,11 +122,8 @@
     <feature name="catalog-core-directorymonitor" version="${project.version}"
              description="Monitors directories to process content files.">
         <feature>apache-commons</feature>
-        <feature>bootflag-api</feature>
         <feature>catalog-transformer-bootflag</feature>
         <bundle>mvn:com.hazelcast/hazelcast/${hazelcast.version}</bundle>
-        <bundle>mvn:org.apache.httpcomponents/httpclient-osgi/${httpclient.version}</bundle>
-        <bundle>mvn:org.apache.httpcomponents/httpcore-osgi/${httpcore.version}</bundle>
```

Comment:
We should add a manual test step to the PR to make sure that CDM isn’t broken by the CDM feature/pom changes